src/rdkafka_admin.c

```diff
@@ -4808,7 +4807,293 @@ void rd_kafka_DescribeAcls(rd_kafka_t *rk,
 
         rd_kafka_q_enq(rk->rk_ops, rko);
 }
+rd_kafka_resp_err_t rd_kafka_DescribeUserScramCredentialsRequest(rd_kafka_broker_t *rkb,
+    const rd_list_t *userlist,
+    rd_kafka_AdminOptions_t *options,
+    char *errstr,
+    size_t errstr_size,
+    rd_kafka_replyq_t replyq,
+    rd_kafka_resp_cb_t *resp_cb,
+    void *opaque){
+        /*
+                DescribeUserScramCredentials Request (Version: 0) => [users] TAG_BUFFER 
+                users => name TAG_BUFFER 
+                        name => COMPACT_STRING
+        */
+        rd_kafka_buf_t *rkbuf;
+        int16_t ApiVersion = 0; 
+        int features;
+        /*Add the correct version of Api*/
+        ApiVersion = rd_kafka_broker_ApiVersion_supported(
+            rkb, RD_KAFKAP_DescribeUserScramCredentials, 0, 0, &features);
+        if (ApiVersion == -1) {
+                rd_snprintf(errstr, errstr_size,
+                            "DescribeUserScramCredentials API (KIP-554) not supported "
+                            "by broker");
+                return RD_KAFKA_RESP_ERR__UNSUPPORTED_FEATURE;
+        }
+
+        int num_users = rd_list_cnt(userlist);
+
+        rkbuf = rd_kafka_buf_new_request(rkb, RD_KAFKAP_DescribeUserScramCredentials, 1,
```

Comment:
Need to use flexver, see same comment on alteruserscramcredentials

Reply:
Sure !